packages/ember-metal/lib/utils.js

```diff
@@ -105,6 +105,14 @@ function intern(str) {
   return str;
 }
 
+export function symbol(debugName) {
+  // TODO: Investigate using platform symbols, but we do not
+  // want to require non-enumerability for this API, which
+  // would introduce a large cost.
+
+  return intern(debugName + ' [id=' + GUID_KEY + Math.floor(Math.random() * new Date()) + ']');
```

Comment:
Are there really platforms where Math.random() is so broken that we need to multiply by `new Date`?